src/components/panel/panel.spec.js

```diff
@@ -1053,6 +1053,125 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('grouping logic:', function() {
+    it('should create a group using the newPanelGroup method', function() {
+      $mdPanel.newPanelGroup('test');
+
+      expect($mdPanel._groups['test']).toExist();
+    });
+
+    it('should create a group using the config option groupName when the ' +
+        'group hasn\'t been created yet', function() {
+          var config = {
+            groupName: 'test'
+          };
+
+          openPanel(config);
+
+          expect($mdPanel._groups['test']).toExist();
+
+          closePanel();
+        });
+
+    it('should add a panel to a group using the addToGroup method', function() {
+      $mdPanel.newPanelGroup('test');
+
+      expect($mdPanel._groups['test']).toExist();
+
+      var panel1 = $mdPanel.create(DEFAULT_CONFIG);
+
+      panel1.addToGroup('test');
+
+      var panelFromGroup = $mdPanel._groups['test'].panels[0];
+
+      expect(panel1).toEqual(panelFromGroup);
+    });
+
+    it('should add a panel to a group using the config option groupName',
+        function() {
+          $mdPanel.newPanelGroup('test');
+
+          expect($mdPanel._groups['test']).toExist();
```

Comment:
This expect doesn't matter. It's not relevant to the given tests (you've already tested that creating a group works). Ditto for all other tests besides the create newPanelGroup one.